model/schema/reference.yaml

```diff
@@ -21,6 +21,7 @@ classes:
       - abstract
       - category
       - citation
+      - curie
```

Comment:
Added because there is a `references.curie` column in the ABC db. Not sure if this was intentionally left out.